content/swagger/responses.yaml

```diff
@@ -134,6 +134,15 @@ NoContent:
       }
     }
   }
+429Error:
+  description: Too many requests
+  x-details: There are too many requests on this endpoint
+  schema:
+    $ref: "#/definitions/Error"
+  examples: {
+    "code": 429,
+    "message": ""
```

Comment:
Not great to have an empty message. I'll coordinate with Christophe to define one.